recipe/meta.yaml

```diff
@@ -17,40 +16,60 @@ build:
 
 requirements:
   host:
-    - python >=3.6,<3.10
-    - poetry
+    - python >=3.6.2,<3.11
+    - poetry-core >=1.0.0
     - pip
   run:
-    - python >=3.6,<3.10
-    - boto3 >=1.16.8,<2.0.0
-    - botocore >=1.19.8,<2.0.0
-    - numpy >=1.18.0,<2.0.0
-    - pandas >=1.1.0,<2.0.0
-    - pyarrow >=2.0.0,<5.1.0
-    - redshift_connector ~=2.0.887
+    - boto3 >=1.20.17,<2.0.0
+    - botocore >=1.23.17,<2.0.0
+    - jsonpath-ng >=1.5.3,<2.0.0
+    - numpy >=1.18.0,<1.22
```

Comment:
I would let conda handle this as the solver should pull in the appropriate version automatically. Or is there a different reason why different versions are restricted to different python versions?

Reply:
AWS Data Wrangler supports Python versions from 3.6 to 3.10. Numpy on the other hand restricts some of its releases to specific Python versions. For example, 1.21.0 is Py38 and above only. If you believe that the conda solver can handle this then no issue here

Reply:
Yes, the solver will pull in the appropriate version for each python version. The only point of failure is if numpy on conda-forge is not correctly assigned to the different python versions but that is an issue for them and would also break a version-specific pin here.